modules/apps/style-book/style-book-web-test/src/testIntegration/java/com/liferay/style/book/web/internal/portlet/action/test/ExportImportStyleBookEntriesMVCResourceCommandTest.java

```diff
@@ -117,12 +117,15 @@ public void testExportImportSingleStyleBookEntry() throws Exception {
 			ServiceContextTestUtil.getServiceContext(
 				_sourceGroup, TestPropsValues.getUserId());
 
+		String themeId = RandomTestUtil.randomString();
+
+		String name = RandomTestUtil.randomString();
+
 		StyleBookEntry styleBookEntry =
 			_styleBookEntryLocalService.addStyleBookEntry(
 				null, TestPropsValues.getUserId(), _sourceGroup.getGroupId(),
-				false, _read("frontend-tokens-values.json"),
-				"Style Book Entry Name", "STYLE_BOOK_ENTRY_KEY",
-				RandomTestUtil.randomString(), serviceContext);
+				false, _read("frontend-tokens-values.json"), name,
+				"STYLE_BOOK_ENTRY_KEY", themeId, serviceContext);
```

Comment:
You replaced `"Style Book Entry Name"` with `RandomTestUtil.randomString()`, can the same be done for `"STYLE_BOOK_ENTRY_KEY"`?